addons/udes_priorities/models/stock_picking.py

```diff
@@ -10,9 +10,12 @@ def _default_priority(self):
         return self.env.ref("udes_priorities.normal").reference
 
     priority = fields.Selection(selection="get_priorities_for_selection", default=_default_priority)
+    u_priority_id = fields.Many2one(comodel_name="udes_priorities.priority")
+    u_priority_sequence = fields.Integer(
+        string="Priority Sequence", compute="_compute_priority_sequence", store=False
```

Comment:
Would `related=u_priority_id.sequence` work instead of compute?

Reply:
No, because an additional requirement (which I have no implemented) is that the priority sequence should not be updated on cancelled or completed pickings, and `related=` doesn't let us filter by a search domain.

Good question though :+1: